CODE_OF_CONDUCT.md

```diff
@@ -1,5 +1,149 @@
+
+## Moderator Moderation Process
+
+Once the moderators get involved, they will follow a documented series of steps and do their best to preserve the well-being of project members. This section covers actual concrete steps.
+
+### Contacting Moderators
+
+Official moderation is done by the Ayo.js Moderation Team.
+
+* Ping [`@ayojs/moderation`](https://github.com/orgs/ayojs/teams/moderation/members) on the GitHub issue.
+* Ping `@moderation` in the [Ayo.js Discord](https://discord.gg/hCgptwH).
+* TODO: email?
+
+#### TODO: what are our official contact channels? How do people find us?
+
+### Further Moderation
+
+If you've already followed the [initial moderation steps](#when-something-happens), these are the steps moderators will take for further moderation, as needed:
+
+  1. Repeat the request to stop.
+  2. If the person doubles down, they will have offending messages removed or edited by a moderator and be given an official warning. The PR or Issue may be locked.
+  3. If the behavior continues or is repeated later, the person will be blocked from participating for 24 hours.
+  4. If the behavior continues or is repeated after the temporary block, a long-term (6-12mo) ban will be used.
+
+On top of this, moderators may remove any offending messages, images, contributions, etc, as they deem necessary. When editing messages, please put the original message behind a `<details>` tag, with an appropriate `<summary>` instead of simply editing in place. This will allow context to remain during conversations even when something has been moderated.
+
+Moderators reserve full rights to skip any of these steps, at their discretion, if the violation is considered to be a serious and/or immediate threat to the health and well-being of members of the community. These include any threats, serious physical or verbal attacks, and other such behavior that would be completely unacceptable in any social setting that puts our members at risk.
+
+Members expelled from events or venues with any sort of paid attendance will not be refunded.
+
+### Who Watches the Watchers?
+
+Moderators and other leaders who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project's leadership. These may include anything from removal from the moderation team to a permanent ban from the community.
+
+Additionally, as a project hosted on GitHub, [its own Codes of Conduct may be applied against moderators of this project](#other-community-standards), externally of this project's procedures.
+
+### Moderation Examples
+
+#### The Best Case
+
+The vast majority of situations work out like this. This interaction is common, and generally positive.
+
+> Alex: "Yeah I used X and it was really crazy!"
+
+> Patt (not a moderator!): "Hey, could you not use that word? What about 'ridiculous' instead?"
+
+> Alex: "oh sorry, sure." -> edits old comment to say "it was really confusing!"
```

Comment:
I know it's under construction. Just point some typo which obvious to me.

"it was really confusing" -> "it was really ridiculous" (or changing the above sentence)

Reply:
@Harrison-M Agree. Shouldn't we also point it out to the sample, or just imply it?